core/crypto/abac/proto/abac.proto

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright IBM Corp. 2016 All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		 http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+
+package protos;
+
+//ABACMetadataEntry is an entry within the metadata that store an attribute name with its respective key.
+message ABACMetadataEntry {
+    string AttributeName = 1;
+    bytes AttributeKey = 2;
+}
+
+//ABACMetadata hold both the original metadata bytes and the metadata required by the Attribute Based Access Control
```

Comment:
Just a clarification: Is the marshaled version of ABACMetadata what we will have to put in the transaction metadata for access control?